fxa-oauth-server/lib/assertion.js

```diff
@@ -2,8 +2,27 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+'use stict';
+
+/* Utilities for verifing signed identity assertions.
+ *
+ * This service accepts two different kinds of identity assertions
+ * for authenticating the caller:
+ *
+ *  - A JWT, signed by one of a fixed set of trusted server-side secret
+ *    HMAC keys.
+ *  - A BrowserID assertion bundle, signed via BrowserID's public key
+ *    discovery mechanisms.
+ *
+ * The former is much simpler and easier to verify, so much so that
+ * we do it inline in the server process.  The later is much more
```

Comment:
[The Scientific Case for Two Spaces After a Period](https://www.theatlantic.com/science/archive/2018/05/two-spaces-after-a-period/559304/).

j/k, I'm trying to train myself out of this because [basically](https://www.quickanddirtytips.com/education/grammar/two-spaces-after-a-period) [the ](https://www.cultofpedagogy.com/two-spaces-after-period/) [entire](https://www.lifehacker.com.au/2018/05/no-you-still-shouldnt-put-two-spaces-after-a-period/) [internet](https://slate.com/technology/2011/01/two-spaces-after-a-period-why-you-should-never-ever-do-it.html) disagrees with my primary-school English teacher on the subject...

Reply:
wow, TIL .

Also, we can keep it as is if it makes your day brighter 🌞